soundfile_build.py

```diff
@@ -42,7 +42,7 @@
 
 typedef int64_t sf_count_t ;
 
-typedef struct SNDFILE_tag SNDFILE ;
+typedef	struct sf_private_tag	SNDFILE ;
```

Comment:
This does not change any behavior, right? Why did you did you change it?

Reply:
It does't change any behavior, just keep update with libsndfile. 
I didn't find `typedef struct SNDFILE_tag SNDFILE ;` in the source of libsndfile, but found `typedef	struct sf_private_tag	SNDFILE` in the source file `include/sndfile.h.in`.